04-Coffee-and-Coding.Rmd

```diff
@@ -0,0 +1,44 @@
+# Coffee and Coding {#CC}
+
+## About Coffee and Coding
+
+The aim of the MoJ Coffee and Coding presentations is to nurture, enable and encourage a vibrant, supportive and inclusive coding community at MoJ. Coffee & Coding provides a regular opportunity for people within MoJ who are interested in coding to:
+
+* Share skills and knowledge
+* Showcase their work and projects
+* Provide advice and guidance to each other
+* Network and get to know each other
+
+Code can be shared from any language — whether R, Python, SQL, or other more specialised languages.
+
+The format is generally 30–40 minutes for a presentation, followed by time for questions. The presentations usually take the form of a demonstration of a tool or technique and/or a show and tell of work done within the department using these methods.
+
+## Recorded presentations
+
+Where possible, Coffee and Coding presentations are recorded to allow people to catch up on talks they may have missed and to provide a useful training resource for colleagues.
+
+These recordings are available on the [Coffee and Coding Microsoft Stream Channel](https://web.microsoftstream.com/channel/f6aa6c5d-e90c-44b7-8ccc-28a318fa0630). Access should be granted to all new starters, but if you cannot access the videos please contact the Coffee and Coding mailbox.
```

Comment:
> Access should be granted to all new starters

Maybe rephrase to:

> We regularly update the access list, but if you cannot access the videos please contact the Coffee and Coding mailbox.

As there's a reasonably high chance new starters won't have access?

Reply:
```suggestion
These recordings are available on the [Coffee and Coding Microsoft Stream Channel](https://web.microsoftstream.com/channel/f6aa6c5d-e90c-44b7-8ccc-28a318fa0630). We regularly update the access list, but if you cannot access the videos please contact the Coffee and Coding mailbox.
```